.gitignore

```
@@ -5,3 +5,4 @@ generated_files/
 .config/
 tests/test_gen/
 config.yaml
+result
```

```suggestion
result
.direnv/
```

If we are also adding a `.envrc` we should also have `.direnv/`(direnv's cashe) in `.gitignore`

---

`use nix` doesn't use direnv's cache, afaik, but `use flake` does. wouldn't be bad to add it either way.